asf_heat_pump_suitability/pipeline/prepare_features/listed_buildings.py

```diff
@@ -29,95 +30,58 @@ def transform_df_EPC_X_and_Y_to_point(
     return gdf
 
 
-def get_filtered_df_listed_buildings(data_source_key: str) -> gpd.GeoDataFrame:
+def transform_gdf_listed_buildings(nation: str) -> gpd.GeoDataFrame:
```

Comment:
Here I cleaned the name as it's not a getter function, and also added some additional preprocessing steps to clean and prepare the geometry data for joining to EPC. I also updated the docstring so it makes sense if you load England or Wales data